vue/sbc-common-components/src/components/SbcFeeSummary.vue

```diff
@@ -47,6 +71,12 @@ export default class SbcFeeSummary extends Vue {
   @Prop({ default: '' })
   private payURL!: string
 
+  @Prop({ default: false })
+  private priority!: boolean
```

Comment:
checked with @pksumesh,
since the query param name in the backend is the same, we can keep it this way.

also its now moved to filingData object rather than a separate prop.